src/convert/transformers/metadataTransformerFactory.ts

```diff
@@ -32,15 +32,19 @@ export class MetadataTransformerFactory {
     const type = component.parent ? component.parent.type : component.type;
     const transformerId = type.strategies?.transformer;
     switch (transformerId) {
-      case TransformerStrategy.Standard:
+      case 'standard':
```

Comment:
why don't you like the `TransformerStrategy.Standard` style reference?  Seems easier for refactors and having 1 place to look for the possible transformers.

Reply:
TS type gives you autocomplete and validates that it's one of the possible values.  If you wanted to change the name everywhere, you'd just change the type.

Reply:
I don't really like enums in TS